x-pack/plugins/observability_solution/observability_ai_assistant_app/public/components/chat/starter_prompts.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { css } from '@emotion/css';
+import { uniq } from 'lodash';
+import { useObservabilityAIAssistantAppService } from '../../hooks/use_observability_ai_assistant_app_service';
+import { nonNullable } from '../../utils/non_nullable';
+
+const starterPromptClassName = css`
+  max-width: 50%;
+  min-width: calc(50% - 8px);
+`;
+
+const starterPromptInnerClassName = css`
+  text-align: center !important;
+`;
+
+export function StarterPrompts({ onSelectPrompt }: { onSelectPrompt: (prompt: string) => void }) {
+  const service = useObservabilityAIAssistantAppService();
+
+  const contexts = service.getScreenContexts();
+
+  const starterPrompts = uniq(
+    [...contexts]
+      .reverse()
```

Comment:
I don't think you want to order this. We cannot depend on order, because components get mounted and unmounted, changing the order of when things are registered

Reply:
Interesting. While working and testing the feature I haven't seen it not working. Can you give an example of when this does not work?

Reply:
What do you expect the order to be based? You can probably fairly easily reproduce this by adding useEffect hooks that register starting prompts, and invalidating the dependencies for one on every render, and noticing that that will always come up on top.